client/src/app/rotation-options/rotation-options.component.html

```diff
@@ -0,0 +1,44 @@
+<div class="rotation-text"
+     *ngIf="dpsService.selfJob === jobConsts.SCHOLAR.name">
+  <span class="rotation-tag">Rotation: </span>
```

Comment:
okay now that i've taken a look, a few thoughts: 

* i think it actually looks pretty good overall. the popup itself looks good.
* it may be good to increase the font size slightly of the "Rotation: blah blah blah" text above the table
* also increase the spacing between this text and the table to match the spacing between the table and the github button

Reply:
oh i guess one thing for the popup is maybe consider making it slightly less wide

Reply:
Shrunk the popup:
![popup](https://user-images.githubusercontent.com/5826548/115646170-ab509300-a2ef-11eb-9297-213dd45835b3.png)

Increased the font size. I tried it with the same spacing as between the table and the footer, but imo it looks weirder that way.  It's too far from the table to be grouped with it, but it's not far enough to be a completely separate thing (like the party picker, which is twice as far).
![Client rotations 24px](https://user-images.githubusercontent.com/5826548/115645790-fddd7f80-a2ee-11eb-825b-9c2b92095071.png)